examples/06-verif-manual/vm-002-beam_stresses_and_deflections.py

```diff
@@ -0,0 +1,188 @@
+# ~~~~~~~~~~~~~~~~~~~~~~
+# Set up the nodes and elements. Create nodes then creating elements
+# bewtween nodes.
+
+# Define nodes
+mapdl.n(1)
+mapdl.n(5, 480)
+mapdl.n(6, 60, "1 $ N", 10, 420, 1)
+mapdl.fill(1, 5)
+mapdl.fill(6, 10)
+print(mapdl.nlist())  # List of the created nodes.
+
+# Define elements
+
+mapdl.e(1, 2, 6)
+mapdl.egen(4, 1, 1)
+print(mapdl.elist())  # List of the created elements.
+
+# Display elements with their nodes numbers.
+
+mapdl.eplot(show_node_numbering=True, line_width=5, cpos="xy") 
+
+###############################################################################
+# Define Boundary Conditions
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Application of boundary conditions.
+
+mapdl.d(2, "UX", lab2="UY") 
+mapdl.d(4, "UY")
+mapdl.nsel("S", "LOC", "Y", 0)
+mapdl.d("ALL", "UZ")
+mapdl.d("ALL", "ROTX")
+mapdl.d("ALL", "ROTY")
+mapdl.nsel("ALL")
+
+###############################################################################
+# Define Distributed Loads
+# ~~~~~~~~~~~~~~~~~~~~~~~~~
+# Application a distributed force of :math:`w = (10000/12) lb/in` 
+# in the y-direction.
+
+w = 10000/12  # (lb/in) - distributed load.
+mapdl.sfbeam(1, 1, "PRES", w)
+mapdl.sfbeam(4, 1, "PRES", w)
+mapdl.finish()
+
+###############################################################################
+# Solve
+# ~~~~~
+# Enter solution mode and solve the system.
+
+mapdl.run("/SOLU")
+out = mapdl.solve()
+mapdl.finish()
+print(out)
+
+result = mapdl.result
+print(result)
+
+
+###############################################################################
+# Post-processing
+# ~~~~~~~~~~~~~~~
+# Enter post-processing. To get the stress and deflection results
+# from the middle node and cross-section of the beam we can use
+# mapdl.get().
+
+mapdl.post1()  # Start Post-Processing
+mapdl.set(1)  # Select first Load Step
+
+# Define Maximum Stress
+
+s_eqv_max_str = (mapdl.get('stress_eqv', 'secr', 2, 's', 'eqv', 'max')).split()
+s_eqv_max = -float(s_eqv_max_str[0])
+
+# Difine deflection at the middle of the beam
+
+mid_node_uy = mapdl.get('disp', 'node', 3, 'u', 'y')
+
+###############################################################################
+# Check Results
+# ~~~~~~~~~~~~~
+# Now that we have the results we can compare the nodal displacement and stress
+# experienced by middle node of the beam to the known stresses -11400 psi and 
+# 0.182 inches of the deflection. 
+
+stress_target = -11400.000
+stress_ratio = s_eqv_max / stress_target
+deflection_target = 0.182
+deflection_ratio = mid_node_uy / deflection_target
+
+output = f"""
+-------------------------- VM3 RESULTS COMPARISON --------------------------
+              |   TARGET   |   Mechanical APDL   |   RATIO   |
+----------------------------------------------------------------------------
+    Stress     {stress_target:.3f}        {s_eqv_max:.3f}         {stress_ratio:.3f}
+    Deflection    {deflection_target:.3f}            {mid_node_uy:.3f}            {deflection_ratio:.3f}
```

Comment:
Use fixed length f-strings here instead of manually controlling the number of spaces.  For example, `{deflection_ratio:10.3f}`.